project/bin/get_maintainer_data

```diff
@@ -57,8 +57,11 @@ SHARED_MAINTENANCE = [
 def get_maintainer_info(repo):
     maintainers = set()
     alumni = set()
-    link = 'https://raw.githubusercontent.com/fluxcd/{}/master/MAINTAINERS'.format(
-        repo)
+    if repo == 'flux2':
+        link = 'https://raw.githubusercontent.com/fluxcd/community/main/CORE-MAINTAINERS'
+    else:
+        link = 'https://raw.githubusercontent.com/fluxcd/{}/master/MAINTAINERS'.format(
```

Comment:
```suggestion
        link = 'https://raw.githubusercontent.com/fluxcd/{}/main/MAINTAINERS'.format(
```

I would think?

Reply:
Yes, I was thinking that too, but `flux` hasn't moved to using `main` just yet.